raft-tech/TANF-app
# Backend/517/admin mfa

### This pull request changes...

Redirects the login to the admin portal to login.gov
Closes #517 


## FUNCTIONALITY
When the user is not logged in to the system and attempts to access the Admin at `/admin/` they will be redirected to the usual login flow to login.gov. Once authenticated, when they navigate to `/admin/` they will see the admin console if they have the proper permissions.


## TO TEST

1. Contact @carltonsmith to get `staff` and `superuser` access
2. Go to https://tdp-frontend.app.cloud.gov/ and ensure that you are _not_ logged in
3. Go to https://tdp-backend.app.cloud.gov/admin/ 
4. If you are redirected to login.gov the change is working correctly. Anything else and it is not.
5. Log in through login.gov
6. Go to https://tdp-backend.app.cloud.gov/admin/
7. You now see the admin console

### This pull request is ready to merge when...

  - OpenAPI  
  - Readme
  - Entity Relationship Diagram (ERD)
